seed-data-scripts/get-errors.js

```diff
@@ -0,0 +1,92 @@
+const axios = require('axios');
```

Comment:
all these scripts should be in their own directories with their own package.json files so you can simply install all the required node modules and run the script in it's own directory, independently.

Reply:
ah yes I simply moved them from api/src/utilities

Reply:
done